### PR TITLE
sst dataset

### PR DESCRIPTION
**Pull request by arunraja-hub**

### CODE REVIEW

File: templates/sst/default/templates.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+
+      {{sentence}} |||
+
+      {{tokens}}'
+    name: tokenization
+    reference: ''
+  5119a0b5-5d82-4401-900a-7fafc1d48ff6: !Template
+    id: 5119a0b5-5d82-4401-900a-7fafc1d48ff6
+    jinja: 'How positive is the sentence below?
```
Comment (srush) [inline templates/sst/default/templates.yaml:16]: Let's say "How positive is the movie review below?"

File: templates/sst/default/templates.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+      Tokens: {{tokens}}
+
+
+      Tree: {{tree}}'
+    name: tokenization_tree
+    reference: ''
+  b15994be-ca57-4924-9af7-fbaa6ee0124b: !Template
+    id: b15994be-ca57-4924-9af7-fbaa6ee0124b
+    jinja: 'Is the sentence below positive?
```
Comment (srush) [inline templates/sst/default/templates.yaml:42]: movie review

File: templates/sst/dictionary/templates.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+dataset: sst
+subset: dictionary
+templates:
+  04da3904-527e-4a94-b92a-93ff9a607736: !Template
+    id: 04da3904-527e-4a94-b92a-93ff9a607736
+    jinja: 'Is the word or phrase below positive?
```
Comment (srush) [inline templates/sst/dictionary/templates.yaml:6]: -> "in the context of a movie review"

File: templates/sst/default/templates.yaml [hunk 4]
```diff
@@ -0,0 +1,59 @@
+dataset: sst
+subset: default
+templates:
+  1af99eb0-0bc7-4a41-aee1-fadedb9bafde: !Template
+    id: 1af99eb0-0bc7-4a41-aee1-fadedb9bafde
+    jinja: 'Tokenize the sentence below:
```
Comment (srush) [inline templates/sst/default/templates.yaml:6]: Tokenization is a bit difficult to define. Let's remove this one.

File: templates/sst/default/templates.yaml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    jinja: 'Tokenize and form a sentence parse tree (formatted as a parent pointer
+      tree) of the sentence below:
+
+
+      {{sentence}} |||
+
+      Tokens: {{tokens}}
+
+
+      Tree: {{tree}}'
```
Comment (srush) [inline templates/sst/default/templates.yaml:37]: I am not sure this one is working. The tree looks a bit odd?

**Review by srush — CHANGES_REQUESTED** [review]

**Review by srush — APPROVED** [review]

File: templates/sst/default/templates.yaml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  5119a0b5-5d82-4401-900a-7fafc1d48ff6: !Template
+    id: 5119a0b5-5d82-4401-900a-7fafc1d48ff6
+    jinja: 'How positive is the movie review below?
+
+      Give a score on a scale from 0 to 1.
+
+
+      {{sentence}} |||
+
+      {{label}}'
```
Comment (srush) [inline templates/sst/default/templates.yaml:13]: I just realized that this label is a fraction. Can you convert it into an integer between 1 through 5

File: templates/sst/default/templates.yaml [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+      {{["no","yes"]
+
+      [0 if label < 0.5 else 1]
+
+      }}'
+    name: sentiment_classification
+    reference: ''
+  c5dc88f9-204f-43be-b09f-f6334215e271: !Template
+    id: c5dc88f9-204f-43be-b09f-f6334215e271
+    jinja: "Form a sentence parse tree (formatted as a parent pointer tree) using\
```
Comment (srush) [inline templates/sst/default/templates.yaml:32]: Can you add spaces in this parse? It seems very hard for a language model to get this.

**Comment by srush** [comment]
Looks good. Not sure why there is a conflict though

**Comment by arunraja-hub** [comment]
opened a new PR https://github.com/bigscience-workshop/promptsource/pull/134 to avoid conflicts
closing this

**Pull request closed without merging**